app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,63 @@
+
+$font-color: #088da5;
+
+body {
+
+  background-color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.